packages/modular-scripts/src/__tests__/check/verifyWorkspaceDependencies.test.ts

```diff
@@ -22,16 +22,12 @@ describe('verifyWorkspaceDependencies', () => {
     expect(error).not.toHaveBeenCalled();
   });
 
-  it('rejects packages not in the "packages" directory', async () => {
+  it('doesn\'t reject packages not in the "packages" directory', async () => {
```

Comment:
I think we need a new test case like the one given in your PR description

Reply:
Not sure what you mean here.

Reply:
I think given your other comments, we don't need a new test case, you can disregard this